crates/collab/src/llm/token.rs

```diff
@@ -22,6 +22,7 @@ pub struct LlmTokenClaims {
     pub github_user_login: String,
     pub is_staff: bool,
     pub has_llm_closed_beta_feature_flag: bool,
+    pub has_predict_edits_feature_flag: bool,
```

Comment:
We need to add a `#[serde(default)]` here so that we don’t break deserialization of any existing tokens when this gets deployed.